third_party/cronet_sample/CMakeLists.txt

```diff
@@ -15,11 +16,17 @@ target_include_directories(${PROJECT_NAME} PRIVATE
 
 )
 
-if (UNIX)
+if (UNIX AND NOT APPLE)
```

Comment:
The `APPLE` keywords maps exactly to macOS similar to dart/flutter, Everything else under UNIX possibly maps to Platform.Linux with some exceptions ofcourse, so I think we can let this be.

I couldn't test this on and ARM mac, it's possible something else has to he added for that, but we'd only know if someone tests it.

Reply:
> The `APPLE` keywords maps exactly to macOS similar to dart/flutter, Everything else under UNIX possibly maps to Platform.Linux with some exceptions ofcourse, so I think we can let this be.

Got it 🐱

> I couldn't test this on an ARM mac, it's possible something else has to be added to CmakeList for it to work, but we'd only know if someone tests it.

Let's see :) Maybe it will be similar to iOS build.